dev.lizardbyte.app.Sunshine.yml

```diff
@@ -0,0 +1,217 @@
+app-id: dev.lizardbyte.app.Sunshine
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+separate-locales: false
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: dev.lizardbyte.app.Sunshine.sh
+
+finish-args:
+  - --device=all
+  - --env=PULSE_PROP_media.category=Manager
+  - --filesystem=~/.config/systemd/user:create
+  - --persist=.config/sunshine
```

Comment:
it doesn't use the standard directories?

Reply:
It does not follow the standard directories (XDG).